parser/helper.h

```diff
@@ -242,6 +242,10 @@ class MK {
         return Send0(loc, T(loc), core::Names::attachedClass(), loc);
     }
 
+    static std::unique_ptr<parser::Node> TBind(core::LocOffsets loc, std::unique_ptr<parser::Node> type) {
```

Comment:
What do you think about calling this `TBindSelf` instead?

Reply:
Done, thanks!